### PR TITLE
feat(route): kindle software updates

### PR DESCRIPTION
**Pull request by IvanWng97**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/amazon/kindle/software-updates
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/kindle/radar.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    'https://www.amazon.com/gp/help/customer/display.html?nodeId=GKMQC26VQQMM8XSW': {
```
Comment (TonyRL) [inline lib/v2/kindle/radar.js:2]:
```suggestion
    'amazon.com': {
```
The key accepts domain name only

File: lib/v2/kindle/radar.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        _name: 'Kindle',
+        '.': [
+            {
+                title: '软件更新',
+                docs: 'https://docs.rsshub.app/program-update.html#kindle',
```
Comment (TonyRL) [inline lib/v2/kindle/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/program-update.html#kindle',
                source: ['/gp/help/customer/display.html'],
```

File: lib/v2/kindle/radar.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+                target: '/kindle/software-updates',
```
Comment (TonyRL) [inline lib/v2/kindle/radar.js:8]: `target: (_, url) => new URL(url).searchParams.get('nodeId') === 'GKMQC26VQQMM8XSW'`...

File: lib/v2/kindle/software-updates.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        link: `${url}?nodeId=${nodeIdValue}`,
+        description: 'Kindle E-Reader Software Updates',
+        item: list.map((item) => ({
+            title: item.title + ' - ' + item.version,
+            description:
+                item.description +
+                art(path.join(__dirname, 'templates/software-description.art'), {
+                    item,
+                }),
+            link: item.link,
```
Comment (TonyRL) [inline lib/v2/kindle/software-updates.js:43]: `list` does not have `link`

File: docs/en/program-update.md [hunk 5]
```diff
@@ -284,6 +284,12 @@ Language
 
 <RouteEn author="Jeason0228" example="/ipsw/index/ipsws/iPhone11,8" path="/ipsw/index/:ptype/:pname/" :paramsDesc="['Fill in ipsws or otas to get different versions of firmware','Product name, `http://rsshub.app/ipsw/index/ipsws/iPod`, if you fill in the iPad, follow the entire iPad series(ptype default to ipsws).`http://rsshub.app/ipsw/index/ipsws/iPhone11,8`, if you fill in the specific iPhone11,8, submit to the ipsws firmware information of this model']"/>
 
+## Kindle
+
+### Software Updates
+
+<RouteEn author="NavePnow" example="/kindle/software-updates" path="/kindle/software-updates"/>
```
Comment (TonyRL) [inline docs/en/program-update.md:291]: Please change path to `/amazon/kindle/software-updates`

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Comment by NavePnow** [comment]
All changes have been resolved. Thanks!

**Pull request merged**